handler.go

```diff
@@ -2,21 +2,19 @@ package main
 
 import (
 	"cmp"
-	"fmt"
+	"embed"
 	"html/template"
 	"net/http"
 	"path"
 	"slices"
 	"strings"
-
-	"go.uber.org/sally/templates"
 )
 
 var (
-	indexTemplate = template.Must(
-		template.New("index.html").Parse(templates.Index))
-	packageTemplate = template.Must(
-		template.New("package.html").Parse(templates.Package))
+	//go:embed templates/*.html
+	templateFiles embed.FS
+
+	templates = template.Must(template.ParseFS(templateFiles, "templates/*.html"))
```

Comment:
Minor: code here doesn't always comply with this, but with a broad name like "templates", we should add a `_` prefix to the unexported global -- since there's a good chance there will be a function with a local variable named 'templates'.

ref: https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_